### PR TITLE
Quiz branch

### PR DESCRIPTION
**Pull request by jatinkatyal13**

### CODE REVIEW

**Comment by championswimmer** [comment]
This pull request **introduces 4 alerts** and **fixes 6** when merging 4d257f893aeac73c2f60cdb114d71dc0ca68384c into e446e9ba6e8834c803017a698be3d0f445647420 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-d4a600961706e4ffc34558d9d0098ccc6878cf31)

**new alerts:**

* 4 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 4 alerts** and **fixes 6** when merging 2e78b4b13cb7846a5e7346ad98c104d0cf751651 into e446e9ba6e8834c803017a698be3d0f445647420 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-7559b7cbdceadcb079cadcb1e58e3d8bde75ed99)

**new alerts:**

* 4 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: app/pods/contests/contest/quiz/index/route.js [hunk 1]
```diff
@@ -1,63 +1,83 @@
...
 
   actions: {
-    didTransition () {
-      // Ember.run.later(() => {
-      //   let objDiv = document.getElementById("qTitle");
-      //   window.scrollTo ({top: Ember.$('#qTitle').offset().top - objDiv.scrollHeight});
-      // },500)
-      // return true;
+    error(err) {
+      console.error(err)
```
Comment (witty123) [inline app/pods/contests/contest/quiz/index/route.js]: dont send errors to console

**Comment by championswimmer** [comment]
This pull request **introduces 4 alerts** and **fixes 6** when merging ab9db062582e5976494dd964ff20f82d62b2abc7 into e446e9ba6e8834c803017a698be3d0f445647420 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-4c15ea6583b65b2593228948332464a67a4d48d7)

**new alerts:**

* 4 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 4 alerts** and **fixes 6** when merging e1f81fa81cd92a1f81bef08ab3247f0360be246c into e446e9ba6e8834c803017a698be3d0f445647420 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-3684707273d0d62697aa3d6e76c43cb753e05d64)

**new alerts:**

* 4 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 4 alerts** and **fixes 6** when merging b5acfb270515304b842fd5883c826d64181cd32f into 78910b22eff2627007c7e97b04eb2f3802780016 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-355ebe4067d4d199a8554dca184eb46b1f6a2542)

**new alerts:**

* 3 for Unused variable, import, function or class
* 1 for Useless assignment to local variable

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 3 alerts** and **fixes 6** when merging 31aa56ff835aa5c564cfa6902907bb2763ae30cd into 6bddf8578626482e21a4282e70134db0cb7eabb5 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-1b1f0ea646730053667587f2e152298ddd62414a)

**new alerts:**

* 3 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 3 alerts** and **fixes 6** when merging 6dd4a7a4e7b9f464bc5f8a33fa5909cc664561a6 into 6bddf8578626482e21a4282e70134db0cb7eabb5 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-432a4d52327e49f3c76d1501429488ddb8558d04)

**new alerts:**

* 3 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 3 alerts** and **fixes 6** when merging edab021868c733ab4a2352970edd7c36efa44879 into 6bddf8578626482e21a4282e70134db0cb7eabb5 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-6894e6d2bce00da35b8c33997a19a9b53ae0cf16)

**new alerts:**

* 3 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by championswimmer** [comment]
This pull request **introduces 3 alerts** and **fixes 6** when merging bc2aec58cf64711e72639ada7b05814d528b5c84 into 6bddf8578626482e21a4282e70134db0cb7eabb5 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/hacker-blocks/rev/pr-c36bb440fc42b9e1fd286b52194297a3470662e0)

**new alerts:**

* 3 for Unused variable, import, function or class

**fixed alerts:**

* 6 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**